html-lists/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <h1>HTML Lists</h1>
+    <h2>Ordered List Example</h2>
+      <ol>
+        <li>Learn to code.</li>
+        <li>Build projects.</li>
+        <li>Get a job.</li>
+      </ol>
```

Comment:
Careful of your indentation here! The `ol` is a sibling to the `h2`, so they should be at the same level of indentation.